paasta_tools/long_running_service_tools.py

```diff
@@ -365,6 +387,14 @@ def validate(self, params: Optional[List[str]] = None,) -> List[str]:
                 f"The following registrations for {service_instance} are "
                 f"invalid: {registrations_str}"
             )
+        if registrations_not_in_smartstack:
+            service_instance = compose_job_id(self.service, self.instance)
+            registrations_str = ", ".join(registrations_not_in_smartstack)
+            error_messages.append(
+                f"Service registrations must exist in the smartstack. "
```

Comment:
super-nit:
```suggestion
                f"Service registrations must exist in smartstack.yaml. "
```

Reply:
Thanks for the suggestion @jfongatyelp and I were trying to figure out the proper term to use. 👍🏽